build_tools/travis/install.sh

```diff
@@ -83,11 +87,7 @@ elif [[ "$DISTRIB" == "ubuntu" ]]; then
     pip install pytest pytest-cov cython==$CYTHON_VERSION
 
 elif [[ "$DISTRIB" == "scipy-dev" ]]; then
-    # Set up our own virtualenv environment to avoid travis' numpy.
-    # This venv points to the python interpreter of the travis build
-    # matrix.
-    virtualenv --python=python ~/testvenv
-    source ~/testvenv/bin/activate
+    make_conda python=3.7
```

Comment:
I now wonder if we should just use `python` here without pinning the version.

Reply:
Anticipating Python 4 ? ) 

I think it doesn't hurt to indicate explicitly for the reader that it tests 3.7 ..